web-elements/src/test/java/com/demoqa/ButtonsTest.java

```diff
@@ -0,0 +1,127 @@
+package com.demoqa;
+
+import com.demoqa.page.ButtonsPage;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+public class ButtonsTest extends BaseTest {
+    @Test
+    public void testPageIsOpenTest(){
+        ButtonsPage buttonsPage=new ButtonsPage(driver);
+        buttonsPage.open("buttons");
+        String currentURL = driver.getCurrentUrl();
+        String expectedURL = "https://demoqa.com/buttons";
```

Comment:
Use BASE_URL + somePath (like BASE_URL + "/buttons") because your base url can be changed and all test that uses hardcoded URL will be failed